landscape/sysinfo/landscapelink.py

```diff
@@ -7,7 +7,7 @@ def register(self, sysinfo):
 
     def run(self):
         self._sysinfo.add_footnote(
-            "Graph this data and manage this system at:\n"
```

Comment:
definitely opinion-based, but I kind of like having the `\n` to make the URL stand out a little more (and avoid possibly wrapping it on narrowing consoles).